py/sentry_relay/processing.py

```diff
@@ -258,9 +258,14 @@ def parse_release(release, json_loads: Callable[[str | bytes], Any] = json.loads
     )
 
 
-def compare_version(a, b):
+def compare_version(a, b, use_build_code=True):
```

Comment:
Wondering if instead of `use_build_code` we should call it something `semver`, e.g. `compare_version(a, b, semver=True)`.